pkgs/misc/drivers/hplip/default.nix

```diff
@@ -219,7 +219,7 @@ pythonPackages.buildPythonApplication {
     license = if withPlugin
       then licenses.unfree
       else with licenses; [ mit bsd2 gpl2Plus ];
-    platforms = [ "i686-linux" "x86_64-linux" "armv6l-linux" "armv7l-linux" ];
+    platforms = [ "i686-linux" "x86_64-linux" "armv6l-linux" "armv7l-linux" "aarch64-linux" ];
```

Comment:
Don't we need to add to hplipPlatforms above?

Reply:
hmm, I think I did miss some places, yeah.